src/App.js

```diff
@@ -1,9 +1,13 @@
 import './App.css';
-
+import { Header } from './components/Header'
+import { NavBar } from './components/Nav-Bar'
+import { AlwaysRender } from './components/Always-Render'
 function App() {
   return (
     <div className="App">
-
+        <Header/>
+        <NavBar/>
+        <AlwaysRender/>
```

Comment:
By convention components like this tend to be named ___container. Like, <ContentContainer/> or <ArticleContainer/> or so on. Definitely your decision, and in my opinion its not a hard and fast rule. The important point here though is that <AlwaysRender/> doesn't tell me anything about its function or what state it may hold.